ui/components/multichain/notification-list-item-icon/notification-list-item-icon.tsx

```diff
@@ -37,6 +40,21 @@ export type NotificationListItemIconProps = {
   badge?: BadgeProps;
 };
 
+const isIpfsURL = (url: string): boolean => {
+  return url.startsWith('ipfs://');
+};
```

Comment:
nit - A bit of duplicate code. Nothing too serious. If you want to clean it up you can.

Reply:
I think you’re right. For now, I have created a utility only for the check against IPFS. By using the rest as a selector, we need to figure out how to handle a possible refactor within the component (maybe the parent component)